assets/src/edit-story/karma/fixture/containers/carousel.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Container } from './container';
+
+/**
+ * The page carousel.
+ */
+export class Carousel extends Container {
+  constructor(node, path) {
+    super(node, path);
+  }
+
+  get pages() {
+    const pageList = this.getByRole('listbox', { name: 'Pages List' });
+    if (!pageList) {
+      return null;
+    }
```

Comment:
Would it make sense to move this to helper function?

Reply:
Done. One actually is easily expressed via the other one.